.github/workflows/dockerpublish.yml

```diff
@@ -22,87 +22,63 @@ env:
 jobs:
   push:
     runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        registry:
+          - docker
+          - github
+
     steps:
       - uses: actions/checkout@v2
 
       - name: Build image
-        run: docker build --pull -f ./.docker/Dockerfile -t "${IMAGE_NAME}" ./.docker
+        run: docker build --pull -f ./.docker/Dockerfile -t "${{ env.IMAGE_NAME }}" ./.docker
```

Comment:
It's not necessary, but I am using the more explicit `${{ env.IMAGE_NAME }}` notation here instead of `${IMAGE_NAME}`.

I can revert, though, if you prefer!

Reply:
No, I also like more explicit expressions. Just didn't know this works.